lib/tasks/dump_numbers.rake

```diff
@@ -0,0 +1,42 @@
+namespace :numbers do
+  desc "dump numbers for R plots"
+  task :dump => :environment do
+    # check whether directory for dump exists or create
+    Dir.mkdir("#{Rails.root}/public/data/plot_data/") unless File.exists?("#{Rails.root}/public/data/plot_data/")    
+    # start with getting users
+    counter = 0
+    File.open("#{Rails.root}/public/data/plot_data/number_users.csv","w"){ |file|
+      User.find_each do |u|
```

Comment:
Instead of `counter` you can do `User.find_each.with_index do |u, i|` and `file.puts("#{i + 1}\t#{u.created_at}")`